docs/discovery.rst

```diff
@@ -174,7 +198,7 @@ to the tool with ``--password <password>``.
             MAC: 28:6C:07:XX:XX:XX
     room1
             Model: yeelink.light.color1
-            IP address: 192.168.XXX.XXX
+            \IP address: 192.168.XXX.XXX
```

Comment:
Why change this?

Reply:
yup, this is accidental. I'll fix it!
Thanks for the feedback!